crates/nu-protocol/src/shell_error.rs

```diff
@@ -51,8 +51,8 @@ pub enum ShellError {
         #[label("value originates from here")] Span,
     ),
 
-    #[error("Pipeline mismatch.")]
-    #[diagnostic(code(nu::shell::pipeline_mismatch), url(docsrs))]
+    #[error("Only supports for specific types.")]
+    #[diagnostic(code(nu::shell::only_supports_this_input_type), url(docsrs))]
```

Comment:
The diagnostic creates a doc link, but `only_supports_this_input_type` isn't in the docs, is it? Maybe go over there and add it.

Reply:
I think it's fine, that will be added when the next version of nushell is published